Patch-Source: https://git.alpinelinux.org/aports/tree/testing/openttd/no-sse4.patch?id=631059433ebe8b46a288746a64d4ab1699be1558
this sets the sources to use sse4 instructions, which is above our baseline.
it also enables sse4 on pure-x86 with our builds, so best to disable
--- a/cmake/FindSSE.cmake
+++ b/cmake/FindSSE.cmake
@@ -12,6 +12,7 @@
     #include <xmmintrin.h>
     #include <smmintrin.h>
     #include <tmmintrin.h>
-    int main() { return 0; }"
+    # die!
+    int main() { *(int *)0 = 0; }"
     SSE_FOUND
 )
